src/Features/SupportFileUploads/FileUploadController.php

```diff
@@ -8,12 +8,14 @@
 
 class FileUploadController implements HasMiddleware
 {
+    public static string $default_middleware = 'web';
```

Comment:
I think this should be an array to match `FilePreviewController`.

```suggestion
    public static array $defaultMiddleware = ['web'];
```